jboss-kie-process-migration/configure.sh

```diff
@@ -16,10 +16,6 @@ cp -r ${ADDED_DIR}/launch/* ${LAUNCH_DIR}
 mkdir -p ${CONFIG_DIR}
 cp -r ${ADDED_DIR}/configuration/* ${CONFIG_DIR}
 
-mkdir -p ${JDBC_DRIVERS}
-link /usr/lib/java/mariadb-java-client.jar ${JDBC_DRIVERS}/mariadb-java-client.jar
-link /usr/share/java/postgresql-jdbc/postgresql.jar ${JDBC_DRIVERS}/postgresql-jdbc.jar
```

Comment:
Here was included Mysql/Mariadb and Postgresql drivers , with the changes PostgreSQL drivers aren't included is it correct ?

Reply:
yes, jdbc drivers are now Quarkus extensions, see: https://quarkus.io/guides/datasource